Extensions/PlatformBehavior/platformerobjectruntimebehavior.ts

```diff
@@ -1143,6 +1143,14 @@ namespace gdjs {
       return this._gravity;
     }
 
+    /**
+     * Get the slope maximun angle of the Platformer Object.
+     * @returns the slope maximun angle.
+     */
+    getSlopeMaxAngle(): float {
```

Comment:
Defining this maybe means that there are also missing expression, condition and actions on this parameter

Reply:
Yeah I agree, but I'm not familiar with the platformer behavior so unsure this is wise exposing this value.
@D8H should we have an expression/condition for this value? (or should it not exist as it is not used?)

Reply:
I don't think it would be useful as it's a setting that must be chosen once and for all.

Reply:
One could want to change it though? If your platformer is a skier that is equipped with sealskin, it could go up slopes that are 30° but without them, only 5° (That's just en example)

Reply:
This property is what tells apart a slope from a wall. Set to less than 50°, it will look broken.